Revised Release Notes and Readme Markdown Files (#27)
iofoundry/ontology: core/README.md

<img src="https://user-images.githubusercontent.com/12449023/166088435-a9fcc4c7-f51d-443a-b1fd-9fe96a204f77.png" >
</p>

# The IOF Core Ontology

Version  1, Release 2022-02

*The IOF is an international standards group that operates under the auspices of the Open Applications Group.*

# Introduction

> The IOF Core Ontology (Core) is a formal ontology that provides a common vocabulary for the industrial domains. Core is a "mid-level" ontology in that it resides at the top of the suite of ontologies the IOF seeks to introduce to fulfill its mission of supporting digital manufacturing by standardizing industrial terminology and improving consistency and interoperability across many operational areas of manufacturing and the product life cycle. As a mid-level ontology, Core contains terms used by, or anticipated to be used by, a plurality of ontologies in the suite. Additionally, as the IOF bases all of its ontologies on a single foundational or top-level ontology – for which the IOF has chosen the Basic Formal Ontology or BFO – the Core ontology contains generic terms found in domains beyond industry that build upon BFO and provide a consistent foundation for introducing industry-specific terminology. 

Author: Changes implemented. Also changed the version to 2023-01

> The IOF Core Ontology (Core) is a formal ontology that provides a common vocabulary for the industrial domains. Core is a "mid-level" ontology in that it resides at the top of the suite of ontologies the IOF seeks to introduce to fulfill its mission of supporting digital manufacturing by standardizing industrial terminology and improving consistency and interoperability across many operational areas of manufacturing and the product life cycle. As a mid-level ontology, Core contains terms used by, or anticipated to be used by, a plurality of ontologies in the suite. Additionally, as the IOF bases all of its ontologies on a single foundational or top-level ontology – for which the IOF has chosen the Basic Formal Ontology or BFO – the Core ontology contains generic terms found in domains beyond industry that build upon BFO and provide a consistent foundation for introducing industry-specific terminology. 

Author: I made these changes in a new version of the file (vs using the "commit suggestion" feature of GitHub. Have another look if you wish.

> The IOF Core Ontology (Core) is a formal ontology that provides a common vocabulary for the industrial domains. Core is a "mid-level" ontology in that it resides at the top of the suite of ontologies the IOF seeks to introduce to fulfill its mission of supporting digital manufacturing by standardizing industrial terminology and improving consistency and interoperability across many operational areas of manufacturing and the product life cycle. As a mid-level ontology, Core contains terms used by, or anticipated to be used by, a plurality of ontologies in the suite. Additionally, as the IOF bases all of its ontologies on a single foundational or top-level ontology – for which the IOF has chosen the Basic Formal Ontology or BFO – the Core ontology contains generic terms found in domains beyond industry that build upon BFO and provide a consistent foundation for introducing industry-specific terminology. 

Reviewer: The beginning looks good. I would still move the Getting Involved section to the top level README. It does not seem to be relevant to core.

> The IOF Core Ontology (Core) is a formal ontology that provides a common vocabulary for the industrial domains. Core is a "mid-level" ontology in that it resides at the top of the suite of ontologies the IOF seeks to introduce to fulfill its mission of supporting digital manufacturing by standardizing industrial terminology and improving consistency and interoperability across many operational areas of manufacturing and the product life cycle. As a mid-level ontology, Core contains terms used by, or anticipated to be used by, a plurality of ontologies in the suite. Additionally, as the IOF bases all of its ontologies on a single foundational or top-level ontology – for which the IOF has chosen the Basic Formal Ontology or BFO – the Core ontology contains generic terms found in domains beyond industry that build upon BFO and provide a consistent foundation for introducing industry-specific terminology. 

Author: Deleted the Getting Involved Section from here since it is now covered in root. Have another look at latest commit and please approve so we can publish.

> The IOF Core Ontology is developed and formalized as an ontology using both first-order logic and version 2 of the Web Ontology Language (OWL). The use of logic ensures that each term is defined in a way that is unambiguous to humans and can be processed by computers. The vocabulary in Core is curated by the Core Working Group and an attempt is made to validate and ground terms utilizing use cases from industry.

Author: I made these changes in a new version of the file (vs using the "commit suggestion" feature of GitHub. Have another look if you wish.

# Contents

> 1. core.rdf – an OWL implementation of the IOF Core Ontology, inclusive of natural language definitions, formalizations in first-order logic, examplyes, and comments, serialized in rdf/xml.

Author: Done

> 1. core.rdf – an OWL implementation of the IOF Core Ontology, inclusive of natural language definitions, formalizations in first-order logic, examplyes, and comments, serialized in rdf/xml.

Author: I made these changes in a new version of the file (vs using the "commit suggestion" feature of GitHub. Have another look if you wish.

1. meta – folder containing IOF-specific annotation properties used to define and formalize ontology terms, also serialized in rdf/xml, and imported by the core ontology. 

    * AnnotationVocabulary.rdf -- this is the released version of the IOF Annotation Vocabulary, dated 2022-09-29

> 1. commons-to-core-mapping – folder containing selected mid-level ontologies from the OMG Commons Library of Ontologies, addressing IOF needs in the topic area of identifiers and designators, as well as mapping files to map imported terms to IOF/BFO and align annotation property usage between IOF and OMG.  

Reviewer 2: ```suggestion
1. commons-to-core-mapping – folder containing selected mid-level ontologies from the OMG Commons Library of Ontologies, providing an option to the IOF and OMG users with an interoperable way to use terms in the topic area of identifiers and designators from both IOF and OMG.  
```

Author: I made these changes in a new version of the file (vs using the "commit suggestion" feature of GitHub. Have another look if you wish.

> 	* Commons Folder -- The various OMG Commons Ontology files adopted by Core in this release, including Collections.rdf, Designators.rdf, TextDatatype.rdf and Identifiers.rdf

Reviewer 2: ```suggestion
	* Commons Folder -- The various OMG Commons Ontology files that are interoperable with IOF Core in this release, including Collections.rdf, Designators.rdf, TextDatatype.rdf and Identifiers.rdf
```

Author: I made these changes in a new version of the file (vs using the "commit suggestion" feature of GitHub. Have another look if you wish.

	* meta folder
	    * MappingAnnotationVocabularyToCommons.rdf -- Maps the somewhat different OMG annotation vocabulary to the IOF annotation vocabulary

> 	* MappingCommonsToIOF.rdf -- maps terms from the adopted OMG Commons ontologies to BFO and/or to IOF

Reviewer 2: ```suggestion
	* MappingCommonsToIOF.rdf -- maps terms from the OMG Commons ontologies to BFO and/or to IOF
```

Author: I made these changes in a new version of the file (vs using the "commit suggestion" feature of GitHub. Have another look if you wish.

1. Catalog-v001.xml – a tooling file for rapid loading of the ontology's files in the Protégé ontology development tool.

# Installing / Getting started

> The minimal setup you need short of reading raw XML is a suitable ontology viewer or editing tool, installed on your client machine. IOF recommends a desktop version of the open-source tool [Protégé](https://protege.stanford.edu/), but other open source and commercial tools are known to work as well. The repository for this ontology includes the necessary files for opening the ontology in Protégé without warnings and errors. The same is not guaranteed for other tools. 

Reviewer 2: ```suggestion
The minimal setup you need short of reading raw XML is a suitable ontology viewer or editing tool, installed on your client machine. IOF recommends a desktop version of the open-source tool [Protégé](https://protege.stanford.edu/), but other open source and commercial tools are known to work as well. The repository for this ontology includes the necessary files for opening the ontology in Protégé without warnings and errors. The same is not guaranteed for other tools. 

To use IOF Core without the need to interoperate with OMG Common ontologies, simply import Core.rdf into your ontology. 

To use IOF Core with OMG Common ontologies, import MappingCommonsToIOF.rdf into your ontology. This indirectly import IOF Core and OMG Common ontologies into your ontology.
```

Author: I made these changes in a new version of the file (vs using the "commit suggestion" feature of GitHub. Have another look if you wish.

# Getting Involved

## Reporting Issues for the Core Ontology and Joining the Discussions

The source for Core and its evolution is managed on GitHub. Please log change requests and bugs for the ontology using the "issues tab" of the GitHub repository for this project. However, if you are a member person or organization of the IOF, we ask that you follow the normal process for reporting change requests and issues via the IOF member portal for the Core Working Group.

Requests and reported bugs are monitored at least bi-weekly during which time you can expect to see a response as to next steps. 

## General Discussions

> The IOF welcomes those organizations and persons who would like to contribute to this and other IOF ontology projects. To start contributing, or to join the general discussions, please [click here](https://oagiscore.atlassian.net/wiki/spaces/IOF/overview). 

Author: By top-level you mean in the README.md file in the Ontology folder and applicable to all IOF ontology projects, right? If so, then it means to remove it from here and later, once someone drafts a global README.md file for all our ontologies, we put it back in there, right?

	* MappingCommonsToIOF.rdf -- maps terms from the OMG Commons ontologies to BFO and/or to IOF
1. Catalog-v001.xml – a tooling file for rapid loading of the ontology's files in the Protégé ontology development tool.

# Installing / Getting started

The minimal setup you need short of reading raw XML is a suitable ontology viewer or editing tool, installed on your client machine. IOF recommends a desktop version of the open-source tool [Protégé](https://protege.stanford.edu/), but other open source and commercial tools are known to work as well. The repository for this ontology includes the necessary files for opening the ontology in Protégé without warnings and errors. The same is not guaranteed for other tools.  

To use IOF Core without the need to interoperate with the OMG Commons Ontologies mentioned earlier, simply import Core.rdf into your ontology. To use both IOF Core and the OMG Commons Ontologies mentioned, only import MappingCommonsToIOF.rdf into your ontology. Doing so indirectly imports IOF Core and the applicable OMG Common Ontologies.

> # Getting Involved

Reviewer: This should be moved to the top level.
